src/Avalonia.Diagnostics/Diagnostics/DevToolsDataFormats.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Avalonia.Input;
+
+namespace Avalonia.Diagnostics;
+
+/// <summary>
+/// Contains data formats related to dev tools.
+/// </summary>
+public static class DevToolsDataFormats
+{
+    // TODO: this name isn't ideal. For instance, it's not a valid UTI for macOS.
+    // We currently have a converter in place in native code for backwards compatibility with IDataObject,
+    // but this should ideally be removed at some point.
+    // Consider using DataFormat.CreateApplicationFormat() instead (breaking change).
```

Comment:
From my understanding, `CreateApplicationFormat` won't work here.
This data format is supposed to be "operating system format" one, and shared between devtools (old and new - both use the same data format) and AvaloniaVS extension.
See  ElementsNavView.axaml.cs#L59 in new devtools and [AvaloniaVS](https://github.com/AvaloniaUI/AvaloniaVS/blob/7bf4941dadec857fcc83d859561559426270f8b1/AvaloniaVS.Shared/IntelliSense/XamlPasteCommandHandler.cs#L42) one.

Reply:
Since it's primary for the Visual Studio extension, macOS is not so big of a problem. Unless Rider will support the same format

Reply:
If we want to have the format work on all platforms, it should not be an "operating system format" since the name would need to be different between OSes (macOS must use an UTI). By using an application format, we can tell Avalonia to generate an appropriate but deterministic name here.

However, the right thing to do here if we consider this format public is probably to have proper documented names per OS for it. Then Rider or other extensions are free to use them.